src/modules/db_redis/db_redis_mod.c

```diff
@@ -119,13 +136,25 @@ int keys_param(modparam_t type, void *val)
 		return db_redis_keys_spec((char *)val);
 }
 
+int sentinels_param(modparam_t type, void *val)
+{
+	return parse_sentinel_config((char *)val);
+}
+
 int mod_register(char *path, int *dlflags, void *p1, void *p2)
 {
 	if(db_api_init() < 0)
 		return -1;
 	return 0;
 }
 
+static void db_redis_timer(unsigned int ticks, void *param)
+{
+	if(shared_time) {
```

Comment:
I am wondering if we need to protect the access to this shared variable with a Kamailio lock. The timer will run indepently and then the individual process might access when its changed.